File: HackerRank/Domains/001 Algorithms/001 Warmup/E - Compare the Triplets [Score If Else]/compare-the-triplets.c
#include <stdio.h>

int main() {
    int n, i, a = 0, b = 0;
    int *arr_a, *arr_b;

    arr_a = (int *)malloc(3 * sizeof(int));
    arr_b = (int *)malloc(3 * sizeof(int));

    for (i = 0; i < 3; i++) {
        scanf("%d", &n);
        arr_a[i] = n;
    }

    for (i = 0; i < 3; i++) {
        scanf("%d", &n);
        arr_b[i] = n;
    }

    for (i = 0; i < 3; i++) {
        a += arr_a[i] > arr_b[i] ? 1 : 0;
        b += arr_a[i] < arr_b[i] ? 1 : 0;
    }

    printf("%d %d", a, b);

    return 0;
}

File: HackerRank/Domains/001 Algorithms/001 Warmup/E - Compare the Triplets [Score If Else]/compare-the-triplets.py
def compare_the_triplets(arr_a, arr_b):
    n = len(arr_a)

    a = sum([1 for i in range(n) if arr_a[i] > arr_b[i]])
    b = sum([1 for i in range(n) if arr_a[i] < arr_b[i]])
    
    print(str(a) + ' ' + str(b))

if __name__ == '__main__':
    arr_a = list(map(int, input().rstrip().split()))
    arr_b = list(map(int, input().rstrip().split()))

    compare_the_triplets(arr_a, arr_b)